report_thesis/src/sections/methodology.tex

```diff
@@ -97,31 +97,58 @@ \subsection{Data Preparation}
+
+While these preprocessing techniques are not an exhaustive list, they represent a diverse set of methods.
+Techniques such as feature selection were not considered to limit the scope of the study and due to time constraints.
+
+For the selection of models, we also had several requirements.
+The models selected for experimentation had to be diverse to ensure that we had enough breadth in our results to make informed decisions about which models should be included in the final pipeline.
+Additionally, these models should be suitable for regression tasks and suitable for \gls{libs} data, as evidenced by their \gls{rmsep} and \gls{rmsecv} in the related literature.
+To further bolster our selection pool, we chose to include models that were in the same family of models as those that showed promise in the literature.
+For example, we chose to include \gls{etr} and \gls{ngboost} on this basis.
+
+In Table~\ref{tab:preprocessing-models} summarizes the preprocessing techniques and models selected for our experimentation:
+
+\begin{table}[ht]
+\centering
+\begin{tabularx}{\columnwidth}{lX} % Define the columns, one fixed-width and one flexible
+\toprule
+\multirow{5}{*}{Preprocessing Techniques:} & Z-Score Normalization \\
+                                           & Min-Max Normalization \\
+                                           & Max Absolute Scaling \\
+                                           & Robust Scaling \\
+                                           & Norm 3 \\
+\midrule
+\multirow{6}{*}{Models:}                   & Partial Least Squares \\
+                                           & Support Vector Regression \\
+                                           & Gradient Boost Regression \\
+                                           & Extra Trees Regression \\
+                                           & Natural Gradient Boosting \\
+                                           & Elastic Nets \\
+\bottomrule
+\end{tabularx}
+\caption{Summary of Preprocessing Techniques and Models}
+\label{tab:preprocessing-models}
+\end{table}
```

Comment:
There are superb arguments in this section for some of the preprocessing methods, but seemingly none for the models we use.

I'd also consider other preprocessing methods you mentioned (PCA, transforms)... preprocessing methods. Perhaps this table could be more exhaustive, showing how we categorize the items into:
- Transformations methods
- Dimensionality reduction methods
- Normalization / Scaling
- Models

Which are the relevant items we search for + the hyperparameters of each.

Ideally, I think we update the table to contain everything, and add rationale for our use of the normalization methods + models. Especially now that we have the background section to back it!